doc/man5/config.pod

```diff
@@ -308,7 +308,7 @@ mentioned above.
  cover several lines \
  by including \\ characters
 
- message = Hello World\n
+ message = Hello World\\n
```

Comment:
No, this isn't correct.

Reply:
look at line 309 above.  are you sure this is wrong?

Reply:
The point is what gets embedded into the conf values. You've just changed an embedded line feed to a backslash followed by `n`...